backend/src/main/java/com/google/sps/data/Participant.java

```diff
@@ -1,48 +1,45 @@
 package com.google.sps.data;
 
+import com.google.common.base.MoreObjects;
 import com.google.common.base.Preconditions;
 import java.time.ZonedDateTime;
 
-/** A user who want to be matched. */
+/** A user who wants to be matched. */
 public final class Participant {
 
-  /** Datastore ID */
-  private final long id;
   /** Google username (ldap) */
   private final String username;
-  /** Time user is starts being available */
-  private final ZonedDateTime startTimeAvailable;
+  /** Time user starts being available */
+  // TODO: make these fields final once Participant and User classes are created
+  private ZonedDateTime startTimeAvailable;
   /** Time user is available until */
-  private final ZonedDateTime endTimeAvailable;
+  private ZonedDateTime endTimeAvailable;
   /** How long user wants to chat */
-  private final int duration;
+  private int duration;
+  /** Id of match in datastore, 0 if never found a match */
+  private long matchId;
   /** Time of submitted form */
   private final long timestamp;
 
   /** Initialize constructor fields */
   public Participant(
-      long id,
       String username,
       ZonedDateTime startTimeAvailable,
       ZonedDateTime endTimeAvailable,
       int duration,
+      long matchId,
       long timestamp) {
-    this.id = id;
     this.username = username;
-
     Preconditions.checkArgument(
         startTimeAvailable.isBefore(endTimeAvailable),
         "Start available time must be before end available time.");
```

Comment:
Just wondering, what happens if this precondition fails?

Reply:
Good question, I just checked and it throws an IllegalArgumentException. https://guava.dev/releases/19.0/api/docs/com/google/common/base/Preconditions.html

Reply:
IllegalArgumentException/mvn test error